subsys/bluetooth/shell/CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ zephyr_library_sources_ifdef(
   media_controller.c
   )
 zephyr_library_sources_ifdef(
-  CONFIG_BT_HAS
+  CONFIG_BT_HAS_PRESET_SUPPORT
```

Comment:
This change doesn't look right: `HAS` can exist without preset support, right?

Reply:
This change is for shell support that has only preset related commands for now

Reply:
Right, my bad. Didn't see that this if for the shell :)